recipe/install_nvcc.sh

```diff
@@ -64,11 +64,20 @@ export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include"
 export CPPFLAGS="\${CPPFLAGS} -I\${CUDA_HOME}/include"
 export CXXFLAGS="\${CXXFLAGS} -I\${CUDA_HOME}/include"
 
+echo "ACTIVATE NVCC"
+env
+
+if [[ "\${CONDA_BUILD:-0}" = "1" ]]
+then
+  CONDA_ENV_SYSROOT=\${PREFIX}/\${HOST}/sysroot
```

Comment:
You could make it `CONDA_ENV_SYSROOT=${PREFIX}/${HOST}/sysroot` (without the escapes) and conda will fix the paths when installing this package.

Reply:
I tried that but after installing nvcc_linux-64 locally, the nvcc activate script contains:
```
CONDA_ENV_SYSROOT=/opt/anaconda1anaconda2anaconda3/x86_64-conda_cos6-linux-gnu/sysroot
```
which seems wrong.

Reply:
This is in `/home/pearu/mconda3/pkgs/nvcc_linux-64-10.1-hf484d3e_7/etc/conda/activate.d/nvcc_linux-64_activate.sh`.
But
```
$ ls -ls /home/pearu/mconda3/envs/test-nvcc2/x86_64-conda_cos6-linux-gnu/sysroot/lib/libcuda.so 
0 lrwxrwxrwx 1 pearu pearu 44 nov    3 19:33 /home/pearu/mconda3/envs/test-nvcc2/x86_64-conda_cos6-linux-gnu/sysroot/lib/libcuda.so -> /home/pearu/cuda/10.1/lib64/stubs/libcuda.so
```
which seems correct.

Reply:
I trust that the `PREFIX` part is fixed but will also the `HOST` part be fixed?

Reply:
conda will replace that prefix with the correct one when that package is installed into a conda environment

Reply:
`HOST` part will not be fixed, but I think that's fine for now.

Reply:
Resolved (modulo assuming that `HOST` part is correct).